src/main/java/by/exadel/internship/service/FormService.java

```diff
@@ -67,14 +81,38 @@ private void uploadFile(MultipartFile file, UUID uuid) {
                             (new File(filePath + File.separator + uuid +
                                     File.separator + file.getOriginalFilename())));
             stream.write(bytes);
-            log.info("Success to upload file, form id: "+ uuid);
+
+            log.info("Success to upload file, form id: " + uuid);
+
             stream.close();
         } catch (IOException e) {
             log.error(e.getMessage());
         }
     }
 
-    public List<Form> getAll() {
-        return formRepository.findAll();
+    public List<FormFullDTO> getAll() {
+
+        MDC.put("className", FormService.class.getSimpleName());
+        log.info("Try to get all forms");
+
+        List<Form> formList = formRepository.findAllWithTimeForCallList();
+
+        System.out.println();
+        for (Form f:formList) {
```

Comment:
Remove it at all or do via logger

Reply:
yaa, a forgot about it, thanks, it was for test. Done.